WooCommerce/Resources/en.lproj/Localizable.strings

```diff
@@ -520,12 +505,6 @@
 /* Exception rule for a coupon. Reads like: All Products excl. 2 Products */
 "All Products excl. %1$@" = "All Products excl. %1$@";
 
-/* No comment provided by engineer. */
-"All Products excl. %d Categories" = "All Products excl. %d Categories";
-
-/* No comment provided by engineer. */
-"All Products excl. %d Products" = "All Products excl. %d Products";
-
```

Comment:
Comes from the unit tests, so shouldn't be sent for translation. 🔥 

https://github.com/woocommerce/woocommerce-ios/blob/0073cca68e5a6bc351b51227780d1ce415ae8bac/WooCommerce/WooCommerceTests/Model/CouponWooTests.swift#L119-L128